org.mixedrealitytoolkit.input/Tests/Runtime/InteractorDwellManagerTests.cs

```diff
@@ -112,4 +112,4 @@ public IEnumerator GazeDwellTest()
         }
     }
 }
-#pragma warning restore CS1591
\ No newline at end of file
+#pragma warning restore CS1591
```

Comment:
Us this just spacing changes?  If so, please revert change

Reply:
Done in 6b5b66af0d305ad9d048bfdd41fbe1de0d4b6710 , resolving.